developer/src/tike/actions/dmActionsDebugger.pas

```diff
@@ -439,12 +457,14 @@ procedure TmodActionsDebugger.actDebugViewElementsUpdate(Sender: TObject);
 
 function TmodActionsDebugger.IsDebuggerInTestMode: Boolean;
 begin
-  Result := (ActiveKmnKeyboardEditor <> nil) and (ActiveKmnKeyboardEditor.DebugForm.UIStatus = duiTest);
+  Result := ((ActiveKmnKeyboardEditor <> nil) and (ActiveKmnKeyboardEditor.DebugForm.UIStatus = duiTest)) or
+    (ActiveLdmlKeyboardEditor <> nil);
```

Comment:
Is there an `(ActiveLdmlKeyboardEditor.DebugForm.UIStatus = duiTest)` to check?

Reply:
No, because the ldml debugger only runs in Test mode -- we don't have line-by-line debugging in 17.0 for ldml keyboards.